learnEcl/2600-project.md

```diff
@@ -2,3 +2,99 @@
 title: Project
 slug: project
 ---
+
+# PROJECT
+
+The PROJECT function processes through all records in the record-set performing the TRANSFORM function on each record in turn. PROJECT result always have the same number of rows as input dataset.
```

Comment:
Technically, PROJECT can result in fewer records if SKIP is used in the TRANSFORM.